iscore_app/match.py

```diff
@@ -0,0 +1,169 @@
+from django.http import HttpResponse
+from django.core import serializers
+
+import json
+from iscore_app.models import Matches, Rankings_list_catagories, Catagories, Ranking_Lists, Draws, Entries
+from iscore_app.modelsApi import *
+
+from json import JSONEncoder
+
+def test1(request):
+    match_generate()
+
+    matchesCollection = Matches.objects.filter(draws__category='Girls Israel').filter(
```

Comment:
hard coded category 'Girls Israel'
refactor each hard-coded string to generic use here and elsewhere